src/countOccurrences.js

```diff
@@ -18,7 +18,9 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  const findLetter = new RegExp(letter, 'gi');
+  const result = phrase.match(findLetter);
```

Comment:
new line before return
```suggestion
  const result = phrase.match(findLetter);
  
```